test/Microsoft.DotNet.Tools.Tests.Utilities/RequiresAspNetCore.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Test.Utilities
+{
+    public class RequiresAspNetCore : FactAttribute
+    {
+        public RequiresAspNetCore()
+        {
+            var  d = new RepoDirectoriesProvider();
```

Comment:
d? Can we just call this repoDirectoriesProvider?

Reply:
fixed